### PR TITLE
#229: Added missing libraries and their documentation.

### PR DESCRIPTION
**Pull request by lulivi**
Checkout #229 .

### CODE REVIEW

**Comment by lulivi** [comment]
Please @iblancasa, take a look at the documentation.

File: resources/cmake/FindRTIConnextDDS.cmake [hunk 1]
```diff
@@ -50,12 +50,14 @@
 #   The CPP API library for Distributed Logger if found.
 # - ``RTIConnextDDS::routing_service_infrastructure``
 #   The infrastructure library for Routing Service if found.
+# - ``RTIConnextDDS::routing_service_assign_transformation``
+#   The assign transformation library for Routing Service if found.
 # - ``RTIConnextDDS::routing_service_c``
-#   The C API library for Routing Service if found (includes rtiroutingservice
-#   and rtirsinfrastructure).
+#   The C API library for Routing Service if found (includes rtiroutingservice,
+#   rtirsinfrastructure and rtirsassigntransf).
```
Comment (iblancasa) [inline resources/cmake/FindRTIConnextDDS.cmake]: This one shouldn't include `rtirsassigntransf`. `rtirsassigntransf` must be independent

File: resources/cmake/FindRTIConnextDDS.cmake [hunk 2]
```diff
@@ -50,12 +50,14 @@
...
 # - ``RTIConnextDDS::routing_service_cpp2``
 #   The CPP2 API library for Routing Service if found (includes
-#   rtiroutingservice and rtirsinfrastructure).
+#   rtiroutingservice, rtirsinfrastructure and rtirsassigntransf).
```
Comment (iblancasa) [inline resources/cmake/FindRTIConnextDDS.cmake]: Same as above

File: resources/cmake/FindRTIConnextDDS.cmake [hunk 3]
```diff
@@ -66,9 +68,39 @@
...
+# - ``RTIConnextDDS::metp``
+#   The METP library if found (nddsmetp).
+# - ``RTIConnextDDS::transport_tcp``
+#   The Transport TCP library if found (nddstransporttcp).
+# - ``RTIConnextDDS::transport_tls``
+#   The Transport TLS library if found (nddstransporttls).
+# - ``RTIConnextDDS::transport_wan``
+#   The Transport WAN library if found (nddstransportwan).
+# - ``RTIConnextDDS::recording_service``
+#   The Recording Service library if found (includes rtirecordingservice,
```
Comment (iblancasa) [inline resources/cmake/FindRTIConnextDDS.cmake]: Trailing white-space

File: resources/cmake/FindRTIConnextDDS.cmake [hunk 4]
```diff
@@ -148,6 +182,54 @@
 #   - ``MONITORING_LIBRARIES``
 #     (e.g., ``MONITORING_LIBRARIES_RELEASE_STATIC``)
 #
+# - ``discovery_static``:
+#   - ``DISCOVERY_STATIC``
+#     (e.g., ``DISCOVERY_LIBRARIES_RELEASE_STATIC``) TODO: add more info??
```
Comment (iblancasa) [inline resources/cmake/FindRTIConnextDDS.cmake]: This is also a low bandwidth plugin

File: resources/cmake/FindRTIConnextDDS.cmake [hunk 5]
```diff
@@ -148,6 +182,54 @@
...
+#
+# - ``nddstls``:
+#   - ``NDDSTLS``
+#     (e.g., ``NDDSTLS_LIBRARIES_RELEASE_STATIC``) TODO: add more info??
```
Comment (iblancasa) [inline resources/cmake/FindRTIConnextDDS.cmake]: For the extra info, take a look into the documentation :P

File: resources/cmake/FindRTIConnextDDS.cmake [hunk 6]
```diff
@@ -148,6 +182,54 @@
...
+#
+# - ``metp``:
+#   - ``METP``
+#     (e.g., ``METP_LIBRARIES_RELEASE_STATIC``) TODO: add more info??
+#
+# - ``transport_tcp``:
+#   - ``TRANSPORT_TCP``
+#     (e.g., ``TRANSPORT_TCP_LIBRARIES_RELEASE_STATIC``) TODO: add more info??
```
Comment (iblancasa) [inline resources/cmake/FindRTIConnextDDS.cmake]: Add also the "<lib> libraries if found"

Comment (lulivi) [reply resources/cmake/FindRTIConnextDDS.cmake]: Isn't that written in the imported targets documentation section? Do you want me to write it also here?

**Pull request merged**